log10/cli/completions.py

```diff
@@ -270,7 +270,15 @@ def download_completions(limit, offset, timeout, tags, from_date, to_date, file)
                     fetched_total += new_data_size
 
                     for completion in new_data:
-                        output_file.write(json.dumps(completion) + "\n")
+                        res = _get_completion(completion["id"])
```

Comment:
`Completions()._get_completions()` returns a list of completions with a compact info, such as last messages instead of full. 
`_get_completion(id)` get the full completion by id with all messages. 
So after we get the list of completions, we do a get for each of the ids for the full completion to save.